### PR TITLE
Feature/4174/logo and UI fixes

### PR DESCRIPTION
**Pull request by wendidy**
https://github.com/dockstore/dockstore/issues/4168, https://github.com/dockstore/dockstore/issues/4174 and the 3 `monitor.ts` failed [non-auth test](https://app.circleci.com/pipelines/github/dockstore/dockstore-ui2/5505/workflows/d30022f6-93c2-4795-83af-cc4106019890/jobs/17871)

Screenshots:
image
image
image

For now, we will leave out the coloured labels like on the mock, since we need to find the "category" tag among them and display this only, there is an issue for this: https://github.com/dockstore/dockstore/issues/4184.

### CODE REVIEW

**Comment by coverbeck** [comment]
> Note: sometimes the number of labels seems too large, as you can see in one of the screenshots it doesn't look as pleasing, is there anything I can do?

Suggest running it by @kcullion , if she's willing and has the time.

File: src/app/organizations/collection/collection.component.html [hunk 1]
```diff
@@ -113,19 +114,32 @@ <h5>
...
+                      <div fxLayout="row wrap" fxLayoutGap="1rem" class="labels">
+                        <span
+                          [ngClass]="entry.entryType == 'workflow' ? 'workflow-label-chip' : 'container-label-chip'"
+                          *ngFor="let label of entry.labels"
+                        >
+                          {{ label }}
+                        </span>
+                        <mat-chip
+                          [ngClass]="entry.entryType == 'workflow' ? 'workflow-label-chip' : 'container-label-chip'"
+                          *ngIf="entry.verified == true"
```
Comment (coverbeck) [inline src/app/organizations/collection/collection.component.html:130]: Make triple equals or get rid of the equals and right-side of the equation entirely.

**Review by coverbeck — APPROVED** [review]

File: src/app/organizations/collection/collection.component.html (hunk 1 above)
Comment (wendidy) [reply src/app/organizations/collection/collection.component.html:130]: Was working with Java and C in the past few days so I completely forgot triple equals exist haha

**Comment by kcullion** [comment]
> > Note: sometimes the number of labels seems too large, as you can see in one of the screenshots it doesn't look as pleasing, is there anything I can do?
> 
> Suggest running it by @kcullion , if she's willing and has the time.

I chatted with Wendy - I suggested just showing the most important labels, like the Verified checkmark and the workflow language. Then if category is implemented, you can show the category labels there (hopefully there will be only one or two of them)

**Comment by wendidy** [comment]
> > > Note: sometimes the number of labels seems too large, as you can see in one of the screenshots it doesn't look as pleasing, is there anything I can do?
> > 
> > 
> > Suggest running it by @kcullion , if she's willing and has the time.
> 
> I chatted with Wendy - I suggested just showing the most important labels, like the Verified checkmark and the workflow language. Then if category is implemented, you can show the category labels there (hopefully there will be only one or two of them)

From our conversation, it seems like right now there isn't a way to pinpoint the category tag from all the labels. Think will leave  all the labels out for now. Also would like to know what the others think

**Comment by kcullion** [comment]
> > > > Note: sometimes the number of labels seems too large, as you can see in one of the screenshots it doesn't look as pleasing, is there anything I can do?
> > > 
> > > 
> > > Suggest running it by @kcullion , if she's willing and has the time.
> > 
> > 
> > I chatted with Wendy - I suggested just showing the most important labels, like the Verified checkmark and the workflow language. Then if category is implemented, you can show the category labels there (hopefully there will be only one or two of them)
> 
> From our conversation, it seems like right now there isn't a way to pinpoint the category tag from all the labels. Think will leave all the labels out for now. Also would like to know what the others think

@wendidy sorry, the category was a new thing that was just proposed, so it would have to be built in as a new term to specify on a workflow/tool. Because there are so many labels, we thought it would be nice to have an overall "category" that someone would use to find similar workflows. The designs show what that could look like throughout the site, but I think it's to be left out for now.

**Review by denis-yuen — APPROVED** [review]

**Pull request merged**